js/date_time.js

```diff
@@ -0,0 +1,18 @@
+const getDateTime = () => {
+  const utilityItems = document.querySelectorAll(".utility__item");
+  const dateArr = new Date().toDateString().split(" ").slice(0, -1);
+  const date = `${dateArr[0]} ${dateArr[2]} ${dateArr[1]}`;
+  const time = new Date()
+    .toTimeString()
+    .split(" ")[0]
+    .split(":")
+    .slice(0, -1)
+    .join(":");
+
+  const timeEl = utilityItems[utilityItems.length - 1];
+  const dateEl = utilityItems[utilityItems.length - 2];
+  timeEl.innerText = time;
+  dateEl.innerText = date;
```

Comment:
This works but quite a bit of calculating. Try toLocalString() combine with split (:)  
- Also if you split into threefunctions, one get get time, one to format time, one to update dom testing will be much easier